js/build.js

```diff
@@ -9,181 +14,153 @@ Fliplet.Widget.instance({
     {
       name: 'dt-content',
       displayName: 'Drag&drop area',
-      // placeholder: '<div class="well text-center" data-view="content">Add components here.</div>'
       placeholder: '<div class="well text-center">Add components here.</div>'
     }
   ],
   render: {
     template: [
-      // '<div class="placeholder"><i class="icon fl-icon-settings"></i>{! placeholder !}</div>',
       '<div class="conditional" data-view="dt-content"></div>'
     ].join(''),
-    beforeReady: function () {
+    beforeReady: function() {
       let element = $(this.$el);
-      Fliplet.Env.get('interact') ? element.addClass('edit') : element.removeClass('edit');
+
+      element.toggleClass('edit', Fliplet.Env.get('interact'));
     },
-    ready: async function () {
+    ready: async function() {
       await Fliplet.Widget.initializeChildren(this.$el, this);
 
+      var result;
+      const userNotLoggedMessage = 'User is not logged in';
+
+      function getType(elem) {
+        if (Array.isArray(elem)) {
+          return 'array';
+        }
+
+        return typeof elem === 'string' ? 'string' : false;
+      }
+
+      function decodeHTMLEntities(str) {
+        var temp = document.createElement('div');
+
+        temp.style.display = 'none';
+        temp.innerHTML = str;
+
+        return temp.textContent || temp.innerText;
+      }
+
+      function getParsedType(elem) {
+        try {
+          let parsedValue = JSON.parse(elem);
+
+          if (Array.isArray(parsedValue)) {
+            return 'array';
+          }
+
+          return false;
+        } catch (error) {
+          return 'string';
```

Comment:
Do you technically want it to return `string` or `getType(elem)` so you can find out the actual type?